internal/pkg/model/schema.go

```diff
@@ -497,3 +497,7 @@ type ToRetireAPIKeyIdsItems struct {
 	// Date/time the API key was retired
 	RetiredAt string `json:"retired_at,omitempty"`
 }
+
+// UpgradeDetails Additional upgrade status details.
+type UpgradeDetails struct {
```

Comment:
should we use the upgradedetails model from the agent-lib here? or it is something different?